src/modules/sensors/parameters.cpp

```diff
@@ -188,6 +188,8 @@ void initialize_parameter_handles(ParameterHandles &parameter_handles)
 	(void)param_find("SYS_CAL_TDEL");
 	(void)param_find("SYS_CAL_TMAX");
 	(void)param_find("SYS_CAL_TMIN");
+
+	(void)param_find("SYS_HAS_BARO");
```

Comment:
Likewise for SYS_HAS_BARO. As soon as commander starts it runs the preflight checks which finds these.

Reply:
done